tools/systemctl

```diff
@@ -74,7 +74,7 @@ SYSTEMCTL_UNITS_COMMON_EXTERNAL="mariadb mysql mysqld                   \
 # SYSTEMCTL_UNITS_COMMON_INTERNAL lists the services delivered by our project
 # TODO: legacy-metrics will be removed, drop it there
 SYSTEMCTL_UNITS_COMMON_INTERNAL="bios tntnet@bios fty-outage fty-metric-store   \
-                                fty-tntnet@bios fty-envvars fty-proxy           \
+                                fty-tntnet@bios fty-envvars
```

Comment:
Is the trailing slash really missing here?

Reply:
seconded, typo to me too

Reply:
But the string would work without the backslashes as well.